### PR TITLE
Primitives separation refactor

### PR DESCRIPTION
**Pull request by patrykks**

### CODE REVIEW

File: classifier_program.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+evauluated_expression = 'lambda IN0: ' + code
+classificator = eval(evauluated_expression, context)
+
+images_dir = '../Dane/COIL20/'
+for filename in os.listdir(images_dir):
+    path_to_image = images_dir + filename 
+    image = Image(path_to_image)
+    print(str(filename) + ',' + str(classificator(image)))
+
+
```
Comment (kroczi) [inline classifier_program.py]: too much newlines

File: image_classification.py [hunk 2]
```diff
@@ -360,8 +371,10 @@ def main():
     pset.addEphemeralConstant("size", lambda: Size(random.randint(3, MIN_WIDTH), random.randint(3, MIN_HEIGHT)), Size)
     pset.addEphemeralConstant("index", lambda: Index(random.randint(0, 7)), Index)
 
-    print (pset.primitives)
-    print (pset.terminals)
+    if DEBUG:
+        print (pset.primitives)
+    if DEBUG:
```
Comment (kroczi) [inline image_classification.py]: why two ifs?

**Review by kroczi — APPROVED** [review]

File: classifier_program.py (hunk 1 above)
Comment (patrykks) [reply classifier_program.py]: Done

File: image_classification.py (hunk 2 above)
Comment (patrykks) [reply image_classification.py]: Done

**Pull request merged**